Userland/Libraries/LibCore/StandardPaths.cpp

```diff
@@ -36,19 +39,25 @@ static Optional<StringView> get_environment_if_not_empty(StringView name)
 
 ByteString StandardPaths::home_directory()
 {
+#if defined(AK_OS_WINDOWS)
+    ByteString path = getenv("USERPROFILE");
+#else
     if (auto* home_env = getenv("HOME"))
         return LexicalPath::canonicalized_path(home_env);
 
     auto* pwd = getpwuid(getuid());
     ByteString path = pwd ? pwd->pw_dir : "/";
     endpwent();
+#endif
     return LexicalPath::canonicalized_path(path);
 }
 
 ByteString StandardPaths::desktop_directory()
 {
+#if !defined(AK_OS_WINDOWS)
     if (auto desktop_directory = get_environment_if_not_empty("XDG_DESKTOP_DIR"sv); desktop_directory.has_value())
         return LexicalPath::canonicalized_path(*desktop_directory);
+#endif
 
     StringBuilder builder;
```

Comment:
probably not necessarily important at this moment, but…

you probably should reach to some windows api to get correct path for desktop directory; pretty much the same for documents, downloads, music, pictures, videos.